chrome/sunset.js

```diff
@@ -68,16 +57,21 @@ Sunset.article_url_ = "https://support.google.com/chrome_webstore/?p=keep_my_opt
 
 
 /**
- * Shows a notification if enough time has passed since the previous one
- * was not shown.
+ * Shows a notification if enough time has passed since the previous one.
+ *
  * @private
  */
 Sunset.maybeShowNotification_ = function() {
   chrome.storage.sync.get(null, function(data) {
+    // Populate the starting date with today's date if it isn't already set.
+    var starting_date = data.start ? new Date(data.start) : new Date();
```

Comment:
If data.start is a random string, starting_date can be invalid. I would test (new Date(data.start)).getTime() instead. This is true iff the date object contains a valid date.